.travis.yml

```diff
@@ -1,4 +1,4 @@
 language: node_js
 node_js:
-  - "6"
-  - "8"
\ No newline at end of file
+  - "8"
+  - "12"
```

Comment:
Why not 10? It's a very generic package, so I think it should be tested against all LTS Node Versions.